src/server/controllers/usersController.js

```diff
@@ -1,12 +1,33 @@
 
 const User = require('../models/user');
+var sesh;
 const createUser = async (req, res) =>{
             
 }
 
-const getUser = async (req, res) => {
+const getUserByEmail = async (req, res) => {
+    const user = await User.find().select('-password').lean()
+    if(!user?.length){
+        return res.status(400).json({message:"No user found"})
+    } 
+    res.json(user)
+}
 
-   
+const verifyUser = async (req, res) => {
+
+    const user = await User.findOne({email: req.body.email, password: req.body.password}).then(user => {   
+        if (user) {
+            req.session.email = user.email
+            console.log(`Found user ${user.email}`)
+            res.status(200).json(user)
+        }
+        
+        else {
+            return res.status(404).json({
+                errors: [{ user: "not found" }],
+              });
+            }   
+    })
 }
```

Comment:
@imran1289-ah , I think that `verifyUser` is what `getUserByEmail` should be. The function code reflects that.
On this file, the `getUserByEmail` function is commented out from the exports.

Feel free to correct me if I'm wrong @AnnaHsu1 or @Resident-png 🙂

Reply:
We decided to separated verifyUser, which is for the login, and getUserbyEmail, which is a  more general route for fetching users. It is not currently used but we decided to leave it here for the future.